core/new-gui/src/app/workspace/component/code-editor-dialog/code-editor-dialog.component.ts

```diff
@@ -43,6 +47,7 @@ export class CodeEditorDialogComponent implements AfterViewInit, SafeStyle, OnDe
   private formControl: FormControl;
   private code?: YText;
   private editor?: any;
+  socket?: any;
```

Comment:
make it private. also, try to give an explicit type instead of `any`.

Reply:
also please rename it to explicitly describe the socket for language server.

Reply:
it is normal for a type to be optionally undefined, you can either use `private languageServerSocket?: Websocket` or `private languageServerSocket: Websocket | undefined`.

Reply:
similarly, if you can fix this type it would be great `private editor?: any;`